configs/arrow.json

```diff
@@ -3,21 +3,15 @@
   "start_urls": ["https://arrow-kt.io"],
   "stop_urls": [
     "https://arrow-kt.io/docs/[0-9]",
-    "https://arrow-kt.io/docs/next",
-    ".*(?<!index\\.html)$"
```

Comment:
Is the goal of removing the `index.html` filter to list the API/functions? 

Adding `.doc-content tr > td:first-child` would do the same without adding duplicates

You'd have something like this

<img width="630" alt="Screenshot 2021-04-05 at 15 03 18" src="https://user-images.githubusercontent.com/20689156/113576806-81754c00-9620-11eb-9bc7-05e9ac76987c.png">
 to the search results.

Reply:
Thanks @shortcuts !! :raised_hands: